src/api/libraryApi.ts

```diff
@@ -38,3 +46,16 @@ export function createLibrary(library: LibraryRequest): Promise<Library> {
     .then((response) => response.json())
     .catch(err => console.log(err));
 }
+
+export function addToLibrary(gameId: string, libraryId: string): Promise<any> {
+    return fetch(homeUrl + librariesBaseUrl  + `${libraryId}/addToLibrary`, {
+        method: "PUT",
+        body: JSON.stringify({
+            "gameId": gameId
+        }),
+        headers: new Headers({'content-type': 'application/json'})
+    })
+        .then((response) => response.statusText)
```

Comment:
Should this return the updated library? I know the backend just returns a string but I wonder if that's actually useful

Reply:
I agree, I think we should do that. I created task: https://gaming-backlog.atlassian.net/browse/GB-78
(I also think we should do it for the delete API I just added, and although it may make more sense to just do this in the tickets that aren't merged now, I'm going to separate it out because it's a bit more work on the backend to return a value for the update call. I can pick this up though ASAP)

Reply:
Yep totally fine to do in a separate ticket. 

I think HTTP DELETE calls typically either return 200 or, even more often, 204 (no content). You're not really supposed to [generate new content in a DELETE call](https://www.rfc-editor.org/rfc/rfc9110.html#name-delete) and I'm realizing now that maybe the new endpoint you created might make more sense as a PATCH or PUT, since we're actually updating the library rather than deleting the game itself. If you agree we can add it to GB-78 that you already made

Reply:
yeah you're totally right, updating that ticket to reflect that